test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -1998,5 +2000,46 @@ namespace iroha {
       executeQuery(query);
     }
 
+    /**
+     * @given initialized storage
+     * @when get pending transactions
+     * @then pending txs storage will be requested for query creator account
+     */
+    TEST_F(QueryExecutorTest, TransactionsStorageIsAccessed) {
+      const auto kPageSize = 100u;
+      auto query = TestQueryBuilder()
+                       .creatorAccountId(account_id)
+                       .getPendingTransactions(kPageSize)
+                       .build();
+
+      EXPECT_CALL(*pending_txs_storage,
+                  getPendingTransactions(account_id, kPageSize, ::testing::_))
```

Comment:
why don't you set expectations on `first_tx_hash` here and below?

Reply:
It is not the main focus of the test. Also, tracking the last parameter will require creation of a custom matcher.